.github/workflows/deploy-vault.yml

```diff
@@ -0,0 +1,62 @@
+name: deploy-vault.ci
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'src/pol-vaults/mainnet.json'
```

Comment:
We may need to expand it for some iVaults.

Reply:
yes, i wanted to test it first for pol vaults.